projects/vdk-plugins/vdk-singer/src/vdk/plugin/singer/sandbox_singer.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from functools import lru_cache
+
+import pkg_resources
+import requests
+from vdk.plugin.singer.singer_command import find_packages_starting_with
+from vdk.plugin.singer.singer_command import is_likely_singer_tap
+
+
+@lru_cache(maxsize=None)
+def console_entry_points():
+    result = dict()
+    for entry_point in pkg_resources.iter_entry_points("console_scripts"):
```

Comment:
Is this loading console_scripts from the singer package? 
could you give an example?

Reply:
Same answer as above, it's just my experimental file.  WIll remove it . Sorry for leaving it in and confusing things.